qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -97,4 +98,6 @@
 from .y import YGate, CYGate
 from .z import ZGate, CZGate
 
+
 from .multi_control_rotation_gates import mcrx, mcry, mcrz
+from .brg import BRGGate
```

Comment:
Very minor, but it would be good to keep the existing alphabetical order here and above if we can.

Reply:
Got it!